purchase_ext_sst/views/purchase_order_views.xml

```diff
@@ -61,6 +61,7 @@
                     </group>
                 </group>
                 <group>
+                    <field name="sale_prediction_amount" />
```

Comment:
Wasn't sure but could it be better placed under remarks field to be consistent with the list view?